sim/firesim-lib/src/main/cc/bridges/tracerv.h

```diff
@@ -98,6 +104,8 @@ class tracerv_t : public bridge_driver_t
   std::string tracefilename;
   std::string dwarf_file_name;
   bool fireperf = false;
+  bool init_ran = false;
+  std::function<void(uint64_t, uint64_t)> callback = NULL;
```

Comment:
Can we name this `on_instruction_received`? Indicate what the callback is providing in the name?